docs/source/tutorial-beginner.rst

```diff
@@ -224,7 +224,7 @@ Now that you have written your first program, you're ready to write a more compl
 ..
 
   a. ``robot.turn_in_place`` directs Cozmo to turn in place.
-  b. ``(degrees(90))`` sets how far he turns in relation to where he is. Cozmo's initial position is assumed to be 0 degrees; he will turn 90 degrees, or directly to his right. The number of degrees goes from 0 - 360, where 0 will not move him and 360 moves him in a complete circle going clockwise. To make Cozmo move counter-clockwise, enter a negative number. For example, entering -90 makes Cozmo turn 90 degrees to the left.
+  b. ``(degrees(90))`` sets how far he turns in relation to where he is. Cozmo's initial position is assumed to be 0 degrees; he will turn 90 degrees, or directly to his left. The number of degrees goes from 0 - 180, where 0 will not move him and 179.99 moves him in almost a semi-circle going anti-clockwise. To make Cozmo turn clockwise, enter a negative number. For example, entering -90 makes Cozmo turn 90 degrees to the right.
```

Comment:
180 seems to be pretty random as to whether it turns clockwise or counter-clockwise, also e.g. 270 turns -90 etc. I imagine that the engine-side code doesn't count for the fact that sin/cos repeat and that it would need to look at magnitude too for full amount of angle required for abs(angle) > ~179.9.